src/Action/GetShortObjectDescriptionAction.php

```diff
@@ -68,7 +69,7 @@ public function __invoke(Request $request)
 
         $object = $admin->getObject($objectId);
 
-        if (!$object && 'html' == $request->get('_format')) {
+        if (!$object && in_array($request->get('_format'), ['html', 'json'])) {
```

Comment:
It will never get to:

```
if ('json' == $request->get('_format')) {
             return new JsonResponse(['result' => [
                 'id' => $admin->id($object),
                 'label' => $admin->toString($object),
             ]]);
         }
```

Why did you add json here?